lxd/db/migration_test.go

```diff
@@ -147,9 +147,11 @@ func TestImportPreClusteringData(t *testing.T) {
 		volumeType := cluster.StoragePoolVolumeTypeImage
 		filters := []db.StorageVolumeFilter{{
 			Type: &volumeType,
+		}, {
+			PoolID: &id,
```

Comment:
This doesn't look quite right? Previously it was filtering by pool id AND volumeType, and now it looks like its filtering by volumeType OR pool id? Am I right?

Reply:
Sorry, I missed the notification for this. Yes, you are correct. I am pushing a new version.